MyPerfectOnboardingApplication/MyPerfectOnboarding.Api/App_Start/WebApiConfig.cs

```diff
@@ -1,18 +1,25 @@
 ﻿using System.Web.Http;
+using System.Web.Http.Routing;
+using Microsoft.Web.Http.Routing;
 
 namespace MyPerfectOnboarding.Api
 {
     public static class WebApiConfig
     {
         public static void Register(HttpConfiguration config)
         {
-            config.MapHttpAttributeRoutes();
+            var constraintResolver = new DefaultInlineConstraintResolver()
+            {
+                ConstraintMap =
+                {
+                    ["apiVersion"] = typeof( ApiVersionRouteConstraint )
+                }
+            };
+            config.MapHttpAttributeRoutes(constraintResolver);
+            config.AddApiVersioning();
 
-            config.Routes.MapHttpRoute(
-                name: "DefaultApi",
-                routeTemplate: "api/{controller}/{id}",
-                defaults: new { id = RouteParameter.Optional }
-            );
+            var json = GlobalConfiguration.Configuration.Formatters.JsonFormatter;
+            json.UseDataContractJsonSerializer = true;
```

Comment:
Please, do not use data contract for JSON serialization, it is like 'drbat se levou rukou za pravym uchem'. WebAPI uses JSON.NET as default serializer, all you need to do is specify SerializerSettings.ContractResolver to camelCase because you want it to transfer your properties to camelCase (Id => id, CreationTime => creationTime), so you can connect your FE to this BE out of the box.